exonum/src/runtime/rust/service.rs

```diff
@@ -235,6 +235,7 @@ impl<'a> AfterCommitContext<'a> {
     }
 }
 
+/// Extended blockchain interface for the service instance authorized as a supervisor.
```

Comment:
Extends ?

Reply:
It's a struct, so I guess `Extended` here is OK.

Reply:
In depends on what `SupervisorExtensions` exactly is. If it's already extended interface than it's ok, but if it extends interface by itself than `Extends` is more appropriate.

Reply:
Well, I still find confusing a verb comment for a struct. Maybe then use "Extensions to the blockchain interface intended for the service instance authorized as a supervisor"?

Reply:
Sounds good as for me. Maybe @Maria-Nosyk can help with it.